types.go

```diff
@@ -1008,8 +1008,8 @@ func (s *ScnLoadData) Uri() string {
 	return Utf16ToString((*uint16)(unsafe.Pointer(s.uri)))
 }
 
-func (s *ScnLoadData) RequestId() C.HREQUEST {
-	return s.RequestId
+func (s *ScnLoadData) RequestId() C.LPVOID {
+	return s.requestId

```

Comment:
This still has some type conversion issues here. I'm not sure which type you're more comfortable returning from this function, but currently the `requestId` is of type `C.HREQUEST`; so, if you're going to return `C.LPVOID` then you'd have to convert the value before returning it. Either that, or you go back to the previous version that returned the `C.HREQUEST` value and keep the following line, from the original the `DataReadyAsync()` function:
```
crequestId := C.LPVOID(requestId)
```
This is actually what I was suggesting in my original comment.

Reply:
https://github.com/sciter-sdk/go-sciter/blob/3b6704eea9f9da3e1730a584a4c694ce6d7844df/include/sciter-x-def.h#L182

https://github.com/sciter-sdk/go-sciter/blob/3b6704eea9f9da3e1730a584a4c694ce6d7844df/include/sciter-x-request.h#L29

Isn't this the same as `C.LPVOID`?

Reply:
Looks the same to me, but Go complains about it for some reason.